src/server/game/Spells/SpellMgr.cpp

```diff
@@ -4903,6 +4903,29 @@ void SpellMgr::LoadSpellInfoCorrections()
 
     // END OF SHADOWFANG KEEP SPELLS
 
+    //
+    // SILVERPINE FOREST SPELLS
+    //
+    // Ride Warhorse
+    ApplySpellFix({ 84164 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_1].Effect &= ~SPELL_EFFECT_APPLY_AURA;
+    });
+
+    // Despawn All Summons (Lordaeron)
+    ApplySpellFix({ 84173 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->Effects[EFFECT_0].RadiusEntry = sSpellRadiusStore.LookupEntry(EFFECT_RADIUS_50000_YARDS);
```

Comment:
why do you want to hit the entire eastern kingdoms with that spell?

Reply:
I need a spell radius further than 800 yards since the summons cover over 700 yards.